optimizely/decision/composite_service_test.go

```diff
@@ -19,81 +19,76 @@ package decision
 import (
 	"testing"
 
-	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
 
 	"github.com/optimizely/go-sdk/optimizely/entities"
 	"github.com/optimizely/go-sdk/optimizely/notification"
 )
 
-func TestGetFeatureDecision(t *testing.T) {
-	mockProjectConfig := new(mockProjectConfig)
-	decisionContext := FeatureDecisionContext{
+type FeatureTestSuite struct {
+	suite.Suite
+	decisionContext    FeatureDecisionContext
+	mockFeatureService *MockFeatureDecisionService
+	testUserContext    entities.UserContext
+}
+
+func (s *FeatureTestSuite) SetupTest() {
+	mockConfig := new(mockProjectConfig)
+	s.decisionContext = FeatureDecisionContext{
 		Feature:       &testFeat3333,
-		ProjectConfig: mockProjectConfig,
+		ProjectConfig: mockConfig,
 	}
-
-	userContext := entities.UserContext{
+	s.mockFeatureService = new(MockFeatureDecisionService)
+	s.testUserContext = entities.UserContext{
 		ID: "test_user",
 	}
+}
 
+func (s *FeatureTestSuite) TestGetFeatureDecision() {
```

Comment:
Do we need test coverage for `NewCompositeService`?

Reply:
Yeah I can add it